cmd/mount.go

```diff
@@ -103,18 +103,47 @@ func (opt *mountOptions) validate(skipNonEmptyMount bool) error {
 		if !os.IsNotExist(err) {
 			return fmt.Errorf("mount directory already exists")
 		}
+
+		if common.IsDirectoryMounted(opt.MountPath) {
+			return fmt.Errorf("directory is already mounted")
+		}
```

Comment:
Actually looking back at this we don't even need to call this, because if the directory exists then it is mounted on Windows and would fail to mount anyway, so no need to do any other checks.